### DISCUSSION TITLE
on remove event

### DISCUSSION THREAD

**Discussion by dairdr** [react-page/react-page#1109] [Q&A] [answered]
Is it possible to know when one plugin is removed from the UI? (when the user clicks on the delete plugin option)

**Comment by macrozone** [comment, marked as answer]
not yes, there is an open issue for it https://github.com/react-page/react-page/issues/1007

whats the use case for that?

**Reply by dairdr** [reply]
let's say I want to remove the image from the server that I just dropped in the `Image` plugin, it'd be great to know if the users removes the plugin from the content editor, so I can call a web service to remove the image file from the server.

**Reply by igorko86** [reply]
@macrozone Hello, do you have any updates about comments from @dairdr above ??
